File: handlers/help.py
from aiogram import Router
from aiogram.types import Message

router = Router()

@router.message(commands=["help"])
async def help_cmd(msg: Message):
    await msg.answer(
        "📘 *Comandos disponíveis:*\n"
        "/start — Iniciar bot\n"
        "/help — Ajuda\n"
        "/status — Status do Sistema\n"
        "/dev — Painel do Desenvolvedor"
    )

File: commands/help.js
export default function helpCommand(bot, msg) {
  const chatId = msg.chat.id;

  bot.sendMessage(
    chatId,
    "ℹ️ Comandos disponíveis:\n/start – inicializa o bot\n/menu – mostra o menu principal\n/help – ajuda"
  );
}
